tools/cldr-apps/src/test/java/org/unicode/cldr/web/TestUserLevel.java

```diff
@@ -0,0 +1,241 @@
+            () -> assertFalse(UserRegistry.userIsExactlyManager(a_vetter), a_vetter + " is exactly manager"),
+            () -> assertFalse(UserRegistry.userIsExactlyManager(a_locked), a_locked + " is exactly manager"),
+            () -> assertFalse(UserRegistry.userIsExactlyManager(a_anonymous), a_anonymous + " is exactly manager"),
+
+            () -> assertTrue(UserRegistry.userCreateOtherOrgs(a_admin), a_admin + " can create orgs"),
+            () -> assertFalse(UserRegistry.userCreateOtherOrgs(a_tc), a_tc + "can create orgs"),
+            () -> assertFalse(UserRegistry.userCreateOtherOrgs(a_manager), a_manager + " can create orgs"),
+            () -> assertFalse(UserRegistry.userCreateOtherOrgs(a_vetter), a_vetter + " can create orgs"),
+            () -> assertFalse(UserRegistry.userCreateOtherOrgs(a_locked), a_locked + " can create orgs"),
+            () -> assertFalse(UserRegistry.userCreateOtherOrgs(a_anonymous), a_anonymous + " can create orgs"),
+
+            () -> assertTrue(UserRegistry.userCanModifyUsers(a_admin), a_admin + " can modify users"),
+            () -> assertTrue(UserRegistry.userCanModifyUsers(a_tc), a_tc + "can modify users"),
+            () -> assertTrue(UserRegistry.userCanModifyUsers(a_manager), a_manager + " can modify users"),
+            () -> assertFalse(UserRegistry.userCanModifyUsers(a_vetter), a_vetter + " can modify users"),
+            () -> assertFalse(UserRegistry.userCanModifyUsers(a_locked), a_locked + " can modify users"),
+            () -> assertFalse(UserRegistry.userCanModifyUsers(a_anonymous), a_anonymous + " can modify users"),
+
+            () -> assertTrue(UserRegistry.userCanEmailUsers(a_admin), a_admin + " can email users"),
+            () -> assertTrue(UserRegistry.userCanEmailUsers(a_tc), a_tc + "can email users"),
+            () -> assertTrue(UserRegistry.userCanEmailUsers(a_manager), a_manager + " can email users"),
+            () -> assertFalse(UserRegistry.userCanEmailUsers(a_vetter), a_vetter + " can email users"),
+            () -> assertFalse(UserRegistry.userCanEmailUsers(a_locked), a_locked + " can email users"),
+            () -> assertFalse(UserRegistry.userCanEmailUsers(a_anonymous), a_anonymous + " can email users"),
+
+            () -> assertTrue(UserRegistry.userCanEmailUsers(a_admin), a_admin + " can email users"),
+            () -> assertTrue(UserRegistry.userCanEmailUsers(a_tc), a_tc + "can email users"),
+            () -> assertTrue(UserRegistry.userCanEmailUsers(a_manager), a_manager + " can email users"),
+            () -> assertFalse(UserRegistry.userCanEmailUsers(a_vetter), a_vetter + " can email users"),
+            () -> assertFalse(UserRegistry.userCanEmailUsers(a_locked), a_locked + " can email users"),
+            () -> assertFalse(UserRegistry.userCanEmailUsers(a_anonymous), a_anonymous + " can email users"),
+
+
+            () -> assertTrue(true, "True is true!")
```

Comment:
low tolerance for ambiguity? no allowance for quantum superpositions?

Reply:
cosmic rays…
no, actually i just was copying and pasting lots of lines, and i wanted to be able to have a trailing comma on the prior lines.